nucleus/settings.py

```diff
@@ -243,6 +243,18 @@
     },
 }
 
-SAML_ENABLE = config('SAML_ENABLE', default=False, cast=bool)
-if SAML_ENABLE:
-    from nucleus.saml.settings import *  # noqa
+OIDC_ENABLE = config('OIDC_ENABLE', default=False, cast=bool)
+if OIDC_ENABLE:
+    AUTHENTICATION_BACKENDS = (
+        'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
```

Comment:
mozilla oidc library does not respect model permissions https://github.com/mozilla/mozilla-django-oidc/issues/176 and we need to do something like this for the django admin to work

https://github.com/mozmeao/snippets-service/commit/1599b29476f1cf9824b376999fc4a189228a42d5

Reply:
Bah! Forgot about that, and I think I even reviewed that one for you :) Thanks!